connect/runtime/src/main/java/org/apache/kafka/connect/runtime/Worker.java

```diff
@@ -324,7 +328,7 @@ public void addTask(ConnectorTaskId id, TaskConfig taskConfig) {
         log.info("Instantiated task {} with version {} of type {}", id, task.version(), taskClass.getName());
 
         // Decide which type of worker task we need based on the type of task.
-        final WorkerTask workerTask;
+        final AbstractWorkerTask workerTask;
```

Comment:
Here's an example where `AbstractWorkerTask` is being used, but there's really no difference between this and the previous version using just `WorkerTask`.